src/cli/cmd/apply/home_manager.rs

```diff
@@ -0,0 +1,74 @@
+use clap::Parser;
+
+use crate::cli::{cmd::parse_release_ref, error::FhError};
+
+pub(super) const HOME_MANAGER_SCRIPT: &str = "activate";
+
+#[derive(Parser)]
+pub(super) struct HomeManager {
+    /// The FlakeHub output reference for the Home Manager configuration.
+    /// References must take one of two forms: {org}/{flake}/{version_req}#{attr_path} or {org}/{flake}/{version_req}.
+    /// If the latter, the attribute path defaults to homeConfigurations.{whoami}.
+    pub(super) output_ref: String,
+
+    #[arg(long, short, env = "FH_APPLY_PROFILE", default_value = "system")]
+    pub(super) profile: String,
```

Comment:
Home manager doesn't use the `system` profile, or even a profile in `/nix/var/nix/profiles/*` at all. In any case we probably shouldn't set this profile ourselves -- since home manager takes pains to do something specific here.

I think for home-manager we shouldn't be installing to a profile at all, but perhaps using a NamedTemporaryDirectory that we write to -- do the action they request -- and then delete.